api/weaveapi.go

```diff
@@ -76,6 +77,22 @@ func (client *Client) Connect(remote string) error {
 	return err
 }
 
+// IsReady returns true if the API server is up and running
+func (client *Client) IsReady() bool {
+	_, err := client.httpVerb("GET", "/status", nil)
+
+	return err == nil
+}
+
+// WaitAPIServer waits until the API server is ready to serve.
+//
+// Might block infinitely.
```

Comment:
Questions:
1. Do we actually want to retry indefinitely? 
2. Will failures be obvious when debugging and/or user support happens?
3. Is there any other information we could leverage to "fail faster" or, alternatively, should we give up after some amount of time?